ISICbalance/ViewControllers/BalanceViewController.swift

```diff
@@ -90,46 +95,57 @@ final class BalanceViewController: BaseViewController {
         balanceTitle.font = UIFont.systemFont(ofSize: 18)
         self.balanceTitle = balanceTitle
         screenStackView.addArrangedSubview(balanceTitle)
-        
+
+        let balanceStack = UIStackView()
+        balanceStack.axis = .horizontal
+        screenStackView.addArrangedSubview(balanceStack)
+
         let balanceLabel = UILabel()
         balanceLabel.textColor = UIColor.theme.textColor
         balanceLabel.adjustsFontSizeToFitWidth = true
         balanceLabel.textAlignment = .center
         balanceLabel.font = UIFont.boldSystemFont(ofSize: 80)
         self.balanceLabel = balanceLabel
-        screenStackView.addArrangedSubview(balanceLabel)
+        balanceStack.addArrangedSubview(balanceLabel)
+
+        let currencyLabel = UILabel()
+        currencyLabel.textColor = UIColor.theme.textColor
+        currencyLabel.adjustsFontSizeToFitWidth = true
+        currencyLabel.textAlignment = .center
+        currencyLabel.font = UIFont.boldSystemFont(ofSize: 80)
+        currencyLabel.text = " \(L10n.Balance.currency)"
```

Comment:
❔ k čemu to tady je? Proč je ten vnitřek interpolovaný?

Reply:
Ten label je kvůli lokalizaci Kč/CZK. A ta interpolace je tam kvůli té mezeře, což není úplně pěkné :D